browsers/node12.8.1-chrome80-ff72/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM cypress/base:12.8.1
+
+USER root
+
+RUN node --version
+RUN echo "force new chrome here!"
```

Comment:
we aren't really forcing a new chrome here tho

Reply:
yup, I just use it to modify if necessary this message and then the Chrome is reinstalled